src/adapters/cdnBucket.ts

```diff
@@ -18,6 +19,7 @@ export async function createCDNBucket({ config }: Pick<AppComponents, 'config'>)
       })
       .promise()
       .then((result) => {
+        if (!!cdnDomain) return cdnDomain + fileName
```

Comment:
No need for `/` or other separator?

Reply:
It is not needed I had it working before but I tried another approach by grabbing the URL that the uploads method returns. Anyways, this last approach wasn't working because the bucket was returning its own URL instead of the CDN one.

Here is how it was the code before these changes -> https://github.com/decentraland/textures-helper/blob/9b2bf5a58efb397cc24e9c3e38c4dd72255bb2ee/src/controllers/handlers/resize-handler.ts#L76